src/Layouts/Partials/AlignItemsPanel.vala

```diff
@@ -70,13 +80,12 @@ public class Akira.Layouts.Partials.AlignItemsPanel : Gtk.Grid {
                                                                   window,
```

Comment:
This indentation is super weird.
Go to another line after the = and leave the attributes inline.

Reply:
Ok

Reply:
What about his indentation?

```vala
var tmp_align_box_button =
    new Akira.Partials.AlignBoxButton (
        window,
        item.action_name,
        item.icon_name,
        item.tooltip_text,
        item.accels
);


```

I think it's more readable, but if you prefer I can put the attributes inline, like this: 

```vala
var tmp_align_box_button =
    new Akira.Partials.AlignBoxButton (window, item.action_name, item.icon_name, item.tooltip_text, item.accels);
```

Reply:
I like it, but put the closing parenthesis after the last attribute, like `item.accels);`